content/en/containers/cluster_agent/admission_controller.md

```diff
@@ -130,11 +130,29 @@ Finally, run the following commands:
 
 ### APM and DogStatsD
```

Comment:
As this page is hard to find, can we had more pointers on our existing doc to the lib injection part?
As an example we could add references in this [section](https://docs-staging.datadoghq.com/eddy/k8-lib-injection/containers/kubernetes/apm/?tab=datadogadmissioncontroller#configure-your-application-tracers-to-emit-traces)

Reply:
+1